### PR TITLE
 Default policies from other lightning nodes

### LINKED ISSUE #12: Default policies from other lightning nodes
This library defines `Policies`: a set of requirements against which channel parameters, proposed to a local node are checked. These policies strictly implement BOLT-2 requirements for processing `open_message` and `accept_message` by a receiving node.

One can read about what these policies are about just by checking doc comments:
https://github.com/LNP-BP/lnp-core/blob/0bd54b92521baa9663a5400a556b391aeafc9de1/src/bolt/channel.rs#L181-L221

Policy is normally provided by a specific node implementation (like LNP Node) to the channel constructor and is read from a user configuration file. However, if the node is just got initiated it will be good to have some preset of the policies which an unexpirienced user can choose from.

For now, we define our own preset for policy configuration here:
https://github.com/LNP-BP/lnp-core/blob/0bd54b92521baa9663a5400a556b391aeafc9de1/src/bolt/channel.rs#L226-L258

However the task is to look/dig into c-lightning, LND and Eclair code and extract their policy defaults and fill them into here:
https://github.com/LNP-BP/lnp-core/blob/0bd54b92521baa9663a5400a556b391aeafc9de1/src/bolt/channel.rs#L261-L274

@grunch I can't assign this task (some strange GitHub bug), so I am asking you to look at it here.

### PR DESCRIPTION
**Pull request by grunch**
This PR add default policies from the main three implementations
but is not finished yet, there is no default value for some values
like `feerate_per_kw_range` on three implementations and
`max_htlc_value_in_flight_msat_min` on lnd, I wrote this on comments

Fix #12

### CODE REVIEW

File: src/channel/bolt/policy.rs [hunk 1]
```diff
@@ -220,17 +220,69 @@ impl Default for Policy {
...
     /// Sets policy to match default policy used in c-lightning
     pub fn with_clightning_defaults() -> Policy {
-        todo!()
+        Policy {
+            to_self_delay_max: 14 * 24 * 6,
+            // I  don't know what is the normal operational range for c-lightning
+            // but I only see values way more than 100 in examples like this
+            // https://github.com/ElementsProject/lightning/blob/335ef3fb69f923f6f30213f68881d10ee6d977ed/doc/PLUGINS.md#openchannel
+            // or tests like this https://github.com/ElementsProject/lightning/blob/master/tests/test_wallet.py#L594
+            //feerate_per_kw_range: 1..100,
```
Comment (dr-orlovsky) [inline src/channel/bolt/policy.rs]: I expect that without this field the code will fail to compile. Can you put here something like `1..1000`?

File: src/channel/bolt/policy.rs [hunk 2]
```diff
@@ -220,17 +220,69 @@ impl Default for Policy {
...
 
     /// Sets policy to match default policy used in LND
     pub fn with_lnd_defaults() -> Policy {
-        todo!()
+        Policy {
+            to_self_delay_max: 14 * 24 * 6,
+            // I am not sure if this is the correct range for LND, even is the correct minimum
+            // I don't find the maximum value
+            // https://github.com/lightningnetwork/lnd/blob/master/lnwallet/chainfee/rates.go#L13
+            // feerate_per_kw_range: 253..?,
```
Comment (dr-orlovsky) [inline src/channel/bolt/policy.rs]: Ibid

File: src/channel/bolt/policy.rs [hunk 3]
```diff
@@ -220,17 +220,69 @@ impl Default for Policy {
...
+        }
     }
 
     /// Sets policy to match default policy used in Eclair
     pub fn with_eclair_defaults() -> Policy {
-        todo!()
+        Policy {
+            to_self_delay_max: 14 * 24 * 6,
+            // Same doubts as with LND and c-lightning about feerate_per_kw_range
+            //feerate_per_kw_range: 1..100,
```
Comment (dr-orlovsky) [inline src/channel/bolt/policy.rs]: Ibid

**Review by dr-orlovsky — CHANGES_REQUESTED** [review]
Thank you for figuring this out! Can you pls rebase on master, where I fixed CI, so we can run tests (`git fetch && git rebase master`)?

File: src/channel/bolt/policy.rs (hunk 1 above)
Comment (grunch) [reply src/channel/bolt/policy.rs]: done

**Review by dr-orlovsky — APPROVED** [review]
utACK 9760b9ae65e14113dbb5dd0adac396980bcf1ccb

Thank you very much!

**Pull request merged**